detector/wordpress.go

```diff
@@ -35,20 +36,37 @@ type WpCveInfos struct {
 
 // WpCveInfo is for wpscan json
 type WpCveInfo struct {
-	ID         string     `json:"id"`
-	Title      string     `json:"title"`
-	CreatedAt  time.Time  `json:"created_at"`
-	UpdatedAt  time.Time  `json:"updated_at"`
-	VulnType   string     `json:"vuln_type"`
-	References References `json:"references"`
-	FixedIn    string     `json:"fixed_in"`
+	ID            string     `json:"id"`
+	Title         string     `json:"title"`
+	CreatedAt     time.Time  `json:"created_at"`
+	UpdatedAt     time.Time  `json:"updated_at"`
+	PublishedDate time.Time  `json:"published_date"`
+	Description   string     `json:"description"` // Enterprise only
+	Poc           string     `json:"poc"`         // Enterprise only
+	VulnType      string     `json:"vuln_type"`
+	References    References `json:"references"`
+	Cvss          Cvss       `json:"cvss"` // Enterprise only
+	Verified      bool       `json:"verified"`
+	FixedIn       string     `json:"fixed_in"`
+	IntroducedIn  string     `json:"introduced_in"`
+	Closed        Closed     `json:"closed"`
```

Comment:
It would be better to know whether it is null or "".
```suggestion
	ID            string     `json:"id"`
	Title         string     `json:"title"`
	CreatedAt     time.Time  `json:"created_at"`
	UpdatedAt     time.Time  `json:"updated_at"`
	PublishedDate time.Time  `json:"published_date"`
	Description   *string     `json:"description"` // Enterprise only
	Poc           *string     `json:"poc"`         // Enterprise only
	VulnType      string     `json:"vuln_type"`
	References    References `json:"references"`
	Cvss          *Cvss       `json:"cvss"` // Enterprise only
	Verified      bool       `json:"verified"`
	FixedIn       string     `json:"fixed_in"`
	IntroducedIn  *string     `json:"introduced_in"`
	Closed        Closed     `json:"closed"`
```

Reply:
Thank you. I have reflected it in the PR.